### PR TITLE
chore: updated screenshots in README.md and added recent events

### PR DESCRIPTION
**Pull request by srinify**
Updated screenshots in README.MD:

gallery

slack_dash

Added some database logos:

firebolt
timescale
rockset

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
...
 [**Organizations Using Superset**](RESOURCES/INTHEWILD.md)
 
+## Why Superset?
+
+Superset is a modern data exploration and data visualization platform. Superset can replace or augment proprietary business intelligence tools for many teams.
+
+Superset provides:
+
+- A **no-code interface** for building charts quickly
+- A powerful, web-based **SQL Editro** for advanced querying
```
Comment (rusackas) [inline README.md:56]:
```suggestion
- A powerful, web-based **SQL Editor** for advanced querying
```

Comment (rusackas) [reply README.md:56]: Actually, not sure if we should capitalize Editor here.

File: README.md [hunk 2]
```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
...
+- A **lightweight semantic layer** for quickly defining custom dimensions and metrics
+- Out of the box support for **nearly any SQL** database or data engine
+- A wide array of **beautiful visualizations** to showcase your data
+metadata browser
```
Comment (rusackas) [inline README.md:60]: This line looks wrong... is this a separate item?

File: README.md (hunk 1 above)
Comment (riahk) [reply README.md:56]: I vote for capitalize! It's a feature name

File: README.md [hunk 3]
```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
...
 
 <kbd><a href="https://superset.apache.org/gallery"><img title="Gallery" src="superset-frontend/src/assets/images/screenshots/gallery.jpg"/></a></kbd><br/>
 
-**View Dashboards**
+**Craft Beautiful, Dynamic Dashboards**
 
 <kbd><img title="View Dashboards" src="superset-frontend/src/assets/images/screenshots/slack_dash.jpg"/></kbd><br/>
 
-**Slice & dice your data**
+**No Code Chart Builder**
```
Comment (rusackas) [inline README.md:76]:
```suggestion
**No-Code Chart Builder**
```
This seems like it should be hyphenated. Maybe. ¯\\\_(ツ)_/¯

File: README.md [hunk 4]
```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
...
 
 <kbd><img title="Slice & dice your data" src="superset-frontend/src/assets/images/screenshots/explore.jpg"/></kbd><br/>
 
-**Query and visualize your data with SQL Lab**
+**Powerful SQL Editor**
 
 <kbd><img title="SQL Lab" src="superset-frontend/src/assets/images/screenshots/sql_lab.jpg"/></kbd><br/>
 
-**Visualize geospatial data with deck.gl**
```
Comment (rusackas) [inline README.md:67]: I'm OK with dropping the section, but for SEO, it seems like we might want to keep the word "geospatial" in the page somewere.

File: README.md [hunk 5]
```diff
@@ -46,55 +46,44 @@ A modern, enterprise-ready business intelligence web application.
...
-- Integration with major
-  authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, etc)
-- The ability to add custom visualization plugins
-- An API for programmatic customization
-- A cloud-native architecture designed from the ground up for scale
-
 ## Supported Databases
 
-Superset can query data from any SQL-speaking datastore or data engine (e.g. Presto or Athena) that has a Python DB-API driver and a SQLAlchemy dialect.
+Superset can query data from any SQL-speaking datastore or data engine (e.g. Presto, Trino, or Athena) that has a Python DB-API driver and a SQLAlchemy dialect.
```
Comment (rusackas) [inline README.md:86]: Should we just make a bullet list of off-the-shelf connectors, for SEO purposes ad/or general completionisim?

**Review by riahk — COMMENTED** [review]
While we're in here, we should remove/update (though I'm not sure it's needed anymore?) the `Dependencies Status` link at the top of the page, as the image is currently broken and the link doesn't go anywhere:
Screen Shot 2021-12-07 at 9 09 14 AM

File: README.md (hunk 5 above)
Comment (srinify) [reply README.md:86]: We have a list here (https://superset.apache.org/docs/databases/installing-database-drivers/). My general feeling is I want to actually reduce the amount of content in the readme and move more of it to the docs / apache site. This seems consistent anyway with most other open source projects have done (e.g. kafka, spark). As a counter point tho, Airflow still has a bunch of content on their readme.

Maybe for now we just do logos, in my next pass I can enumerate the full list?

Comment (rusackas) [reply README.md:84]:
```suggestion
Superset can query data from any SQL-speaking datastore or data engine (Presto, Trino, Athena, [and more](https://superset.apache.org/docs/databases/installing-database-drivers/)) that has a Python DB-API driver and a SQLAlchemy dialect.
```
Let's use the link, then!

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**